### DISCUSSION TITLE
zshrc is getting ignored from ohmyzsh

### DISCUSSION THREAD

**Discussion by WWology** [community/community#4859] [Codespaces]
```zsh
export ZSH="/root/.oh-my-zsh"

ZSH_THEME="cloud"

plugins=(git zsh-syntax-highlighting zsh-autosuggestions)

source $ZSH/oh-my-zsh.sh

alias zshconfig="code /root/.zshrc"
alias ohmyzsh="code /root/.oh-my-zsh"
```

I've put this into my devcontainer

inside Dockerfile
`COPY .zshrc /root/.zshrc` 

and i've already confirmed that inside the container /root/.zshrc is indeed the content of the .zshrc that i pasted

but for some reason, the terminal inside vscode dev are still not using the theme or using the plugin i've specified in the zshrc

is there anyway i can fix this?

**Comment by Chuxel** [comment]
Can you post the rest of the Dockerfile? Depending on what you're using as a base image, OMZ may or may not be there.
